ts/components/wallet/SlidedContentComponent.tsx

```diff
@@ -0,0 +1,48 @@
+import { Content, View } from "native-base";
```

Comment:
Could you leave a comment on what this component does?

Reply:
updated within https://github.com/pagopa/io-app/pull/1628/commits/ad192429d9f5bfc224de114b34a9569a60e3a9a2